packages/common/src/ArcGISContextManager.ts

```diff
@@ -220,6 +259,37 @@ export class ArcGISContextManager {
     return this._context;
   }
 
+  /**
+   * Serialize the context into a string that can be stored
+   * in a browser's local storage or server side session
+   *
+   * @returns encoded string representation of the context
+   */
+  serialize(): string {
+    const state: Partial<IArcGISContextManagerOptions> & { session?: string } =
+      {};
+    if (this._authentication) {
+      state.session = this._authentication.serialize();
+    }
+    if (this._portalUrl) {
+      state.portalUrl = this._portalUrl;
+    }
+    if (this._portalSelf) {
+      state.portal = this._portalSelf;
+    }
```

Comment:
why not serialize this as `state.portalSelf`?

Reply:
mainly b/c of `IArcGISContextManagerOptions.portal` is what it maps into